CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changelog
 
+## 0.44.0      2022-06-28
+
+* Add optional normalization of sensitive fields in EXPLAIN plans
+  - Introduces new "filter_query_sample = normalize" setting that normalizes
+    expression fields in the EXPLAIN plan ("Filter", "Index Cond", etc) using
+    the pg_query normalization logic. Unknown EXPLAIN fields are discarded when
+    this option is active.
+  - Turning on this setting will also cause all query samples (whether they
+    have an EXPLAIN attached or not) to have their query text normalized
+    and their parameters marked as `<removed>`.
+  - This setting is recommended when EXPLAIN plans may contain sensitive
+    data that should not be stored. Please verify that the logic works
+    as expected with your workload and log output.
+* Be more accepting with outdated pg_stat_statements versions
+  - With this change, its no longer required to run
+    "ALTER EXTENSION pg_stat_statements UPDATE" in order to use
+    the collector after a Postgres upgrade
+  - The collector will output an info message in case an outdated
+    pg_stat_statements schema is in use
```

Comment:
If an outdated *version* is in use, no? Separately, I think we also handle the extension installed in a different schema?

Reply:
Yep, good point.